site/docsource/Installation.adoc

```diff
@@ -16,7 +16,7 @@ is installed, run the following command:
 
 [source,sh]
 ------------------------------
-npm install --save bs-platform
+npm install --save-dev bs-platform
```

Comment:
Ah, no, typically TypeScript would spit out a `.d.ts` type declaration file, which would live along side the built Javascript and be included in the npm package. Javascript users wouldn't care about it, but it'd exist for the TypeScript consumers.